service/worker/scheduler/activities.go

```diff
@@ -28,6 +28,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"go.temporal.io/server/common/util"
```

Comment:
move this into the section with server imports

Reply:
weird that make lint-code hasn't done that. Done